DemoApp/README.md

```diff
@@ -1,39 +1,90 @@
-# How to build and test the DemoApp
+### Building and Testing the DemoApp
 
-## Prerequisites
+#### Prerequisites
 
-Since `yarn link/unlink` does not work with react-native. Install [yalc](https://www.npmjs.com/package/yalc) globally. This is a tool that allows you to test local packages without publishing them to npm.
+Since `yarn link/unlink` doesn't work with React Native, you need to install
+[yalc](https://www.npmjs.com/package/yalc) globally. Yalc is a tool that enables
+you to test local packages without publishing them to npm.
 
-// from the root of the project
-// N.B these commands will need to be re-run if you make any changes to the SDK
+#### To install and publish the SDK using yalc, follow these steps:
+
+1. Navigate to the root of the RNTureLayerPaymentsSDK project:
+
+```sh
 cd RNTureLayerPaymentsSDK
-yarn build
-npx yalc publish
+```
+
+2. Publish the project using yalc:
 
-// you may be prompted to install a yalc package if so then accept the installation and rerun the above yalc command
+```sh
+npx yalc publish
 ```
```

Comment:
Should we also mention that you need to do:
```
yarn install
yarn build
```

Reply:
100%. Will add this in.